mlir/include/mlir/Dialect/GPU/GPUOps.td

```diff
@@ -946,4 +946,43 @@ def GPU_BFConvertOp :
   let printer = [{ return ::print(p, *this); }];
   let verifier = [{ return ::verify(*this); }];
 }
+
+// buffer load
+def GPU_MubufLoadOp :
```

Comment:
Seems like this should be promoted eventually. Should we name it more generically, GPU_BufferLoadOp.

Reply:
Let's do it after june demo when and we have a solid plan about upstreaming.